README.md

```diff
@@ -52,6 +52,13 @@ groups:
         severity: critical
       annotations:
         description: "Unable to find or parse expiry for {{ $labels.domain }}"
+    - alert: DomainMetricsAbsent
+      expr: absent(domain_expiration) > 0
+      for: 1h
```

Comment:
By default domain_exporter checks every 12 hours, so I think this should be 12h -- otherwise it will alert until it checks again.

Reply:
Actually this is meant more for kicking off of no metrics were received for an hour (eg. Someone accidentally uninstalled the deployment etc)

Reply:
Ah, gotcha.

Reply:
Dead mans switch, LGTM.